packages/server/src/manifest-generator.ts

```diff
@@ -39,37 +36,48 @@ module.exports = {
   children: children,
 }
 `
-
   yield mkdir(path.dirname(options.destinationPath), { recursive: true });
   yield writeFile(options.destinationPath, manifest);
 }
 
-export function* createManifestGenerator(options: ManifestGeneratorOptions): Operation {
-  if(options.watch) {
-    let watcher = chokidar.watch(options.files, { ignoreInitial: true });
+export const manifestGenerator: Service<ManifestGeneratorOptions> = function *(options) {
+  let { atom, files, mode } = options;
 
-    yield ensure(() => watcher.close());
+  assert(atom, 'No atom in manifestGenerator options');
```

Comment:
If an atom is required, why not make it required at the type level?

Reply:
atom is now required